.github/workflows/build-image.yml

```diff
@@ -0,0 +1,16 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Will it be possible to support Matrix builds here?
Can we build this Dockerfile for other OSes as well. (At least `ubuntu-latest` and `ubuntu-20.04`

Reply:
I guess for windows etc, we need different `Dockerfile`. I leave this optional and can be done in next PR too.

Reply:
Yes. It is possible as well

Reply:
We can add windows Dockerfile in new PR (probably we need to shift these Dockerfiles to a folder) Last time I tried multiple dockerfiles it caused issues.
For matrix build, I have hint in below comment. I guess that can do it.